graphalyzer-server/src/app.py

```diff
@@ -36,7 +36,7 @@ def load_edges(graphlocation: str, graphid: str):
     from py2neo import Graph
     from py2neo.packages.httpstream import http
     # Increase timeout because it can take a really long time.
-    http.socket_timeout = sys.maxsize
+    http.socket_timeout = 9999
```

Comment:
Arbitrary, the default timeout is 30s so pretty much all loads fail with that. There is a potential issue if neo4j goes down after accepting the initial request.